src/ErrorHandler.php

```diff
@@ -135,7 +135,8 @@ public function register(): void
                 return true;
             }
 
-            $backtrace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
+            $backtrace = debug_backtrace();
```

Comment:
```suggestion
            $backtrace = debug_backtrace(0);
```

Reply:
Why?

Reply:
Omits unused index "object" and populates index "args". See https://www.php.net/manual/en/function.debug-backtrace.php